packages/SATSolversForOcaml/SATSolversForOcaml.0.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+name: "SATSolversForOcaml"
```

Comment:
any chance the capitalisation could be made more consistent, or dashes/lowercase used? The language name is "OCaml", so "SATSolversForOCaml" or "ocaml-sat-solvers" would be more natural.